androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/WootricRemoteRequestTask.java

```diff
@@ -111,7 +111,8 @@ protected void onPostExecute(String response) {
             decodedString = new String(response.getBytes("ISO-8859-1"), "UTF-8");
             onSuccess(decodedString);
         } catch (UnsupportedEncodingException e) {
-            e.printStackTrace();
+            onSuccess(response);
+        } catch (NullPointerException e) {
```

Comment:
I've checked the other RequestTasks which extend from `WootricRemoteRequestTask`, all the ones who implement `onSuccess` check if `response != null`